### DISCUSSION TITLE
add serial numbers

### DISCUSSION THREAD

**Discussion by Muhammad-Owais-Warsi** [remoteintech/remote-jobs#1833] [Ideas]
Before each job we can add serial numbers. In future it may help in pagination and stuff.

**Comment by dougaitken** [comment]
Can you elaborate on your idea here please @Muhammad-Owais-Warsi. Any real life examples you could provide?

**Reply by Muhammad-Owais-Warsi** [reply]
Hey @dougaitken,

I was thinking of adding serial numbers to each job listing. This could make navigation easier and help with pagination. Currently, as seen [here](https://remoteintech.company/?utm_campaign=GitHub+bio&utm_source=github&utm_medium=bio&utm_content=textlink), the jobs are listed without numbers. I raised this issue [here](https://github.com/remoteintech/remote-jobs/issues/1834), and I understand the maintainers are considering it for the future.

Thanks :)

**Reply by dougaitken** [reply]
I'm the core maintainer of this project, that's why I asked.

I'm afraid I'm not understanding the use case - each company has a unique page for the company profile linked from the main homepage. There isn't any pagination needed as it's one long list. I do t recognise serial numbers being used in any way so that's why I was asking for examples to reference.

Oh - I see you mention "job listing" - to be clear, this project isn't to list jobs, it's to list companies that hire remotely. It's a static reference rather than a traditional job board.

**Reply by Muhammad-Owais-Warsi** [reply]
Ahh I understand. Thanks for the clarification @dougaitken

**Reply by Muhammad-Owais-Warsi** [reply]
Really sorry for misunderstanding..
